BookingsApi/BookingsApi.IntegrationTests/Api/V2/HearingParticipants/UpdateHearingParticipantsV2Tests.cs

```diff
@@ -284,4 +287,48 @@ public async Task should_return_validation_errors_when_flat_structure_hearing_ro
         var validationProblemDetails = await ApiClientResponse.GetResponses<ValidationProblemDetails>(result.Content);
         validationProblemDetails.Errors[$"{nameof(request.NewParticipants )}[0]"].Should().Contain($"Invalid hearing role [{hearingRoleName}]");
     }
+    
+    [Test]
+    public async Task should_remove_a_participant_from_the_confirmed_hearing()
```

Comment:
Could you add tests for the `GetConfirmedHearingsByUsernameForToday`, please? I have a strange feeling this change has changed the behaviour of the endpoint. Judge or no judge, this should still return hearings that are confirmed

Reply:
Cool I will add that